app/controllers/emails_controller.rb

```diff
@@ -54,15 +54,22 @@ def save_template
       flash.now[:alert] = t('.saving_failed')
     end
     @event = Event.find(params[:event_id])
-    @templates = EmailTemplate.with_status(get_status)
+    @templates = EmailTemplate.with_status(get_email_template_status)
 
     render :email
   end
 
-  def get_status
+  def get_email_template_status
     params[:status] ? params[:status].to_sym : :default
   end
 
+  def get_corresponding_application_letter_status
+    return :accepted if params[:status] == "acceptance"
+    return :rejected if params[:status] == "rejection"
+    # default value
+    return :accepted
```

Comment:
I think a case statement would be slightly better to read (and you get rid of the comment).